src/lug/Window/Unix/WmHints.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+namespace lug {
+namespace Window {
+namespace priv {
+
+#define MWM_HINTS_FUNCTIONS     (1L << 0)
```

Comment:
Use `constexpr` instead

`constexpr long MWMHintsFunctions = 1L << 0`

Same for all the defines below